src/gui/creds/webflowcredentialsdialog.cpp

```diff
@@ -149,6 +151,12 @@ void WebFlowCredentialsDialog::customizeStyle()
     // HINT: Customize dialog's own style here, if necessary in the future (Dark-/Light-Mode switching)
 }
 
+void WebFlowCredentialsDialog::setVisible(bool visible)
```

Comment:
Likely would be better to go through the `showEvent()` handler, otherwise we're doomed to miss cases of visibility changing.